### ISSUE TITLE
Name should be changeable when getting a trustlines request

### ISSUE THREAD

**Issue by berndbohmeier** [trustlines-network/feedback#170]
### Basic information:

* OS (Android / iOS / both): 
Android
* Version: 
1.1.0-dev.5
### Issue description:
When I get a Trustlines request from someone I dont have in my contact book I should be able to change the name right there.

**Comment by Kisgus** [comment]
I agree, but I think having messaged the username that the other party selected is more important
